packages/contracts-bedrock/contracts/L1/SystemConfig.sol

```diff
@@ -77,6 +132,11 @@ contract SystemConfig is OwnableUpgradeable, Semver {
         emit ConfigUpdate(VERSION, UpdateType.GAS_CONFIG, data);
     }
 
+    /**
+     * @notice Updates the L2 gas limit.
+     *
+     * @param _gasLimit New gas limit.
```

Comment:
Thoughts on having a max gas limit set at deploy time and requiring that the new gas limit is less than or equal to the max gas limit? Would prevent a runaway sequencer

Reply:
Eh, I think that's complex. Multisig has other ways it could mess with the system.